src/test/datascience/variableexplorer.functional.test.tsx

```diff
@@ -271,7 +269,8 @@ myTuple = 1,2,3,4,5,6,7,8,9`;
                 openVariableExplorer(wrapper);
 
                 await addCodeImpartial(wrapper, 'a=1\na');
-                await addCodeImpartial(wrapper, basicCode, true, 1);
+                // Variables are fetched in chunks of 5, we have six total here (including sys) so we need two variable fetches
+                await addCodeImpartial(wrapper, basicCode, true, 2);
```

Comment:
So the fix was easy, but the why was a bit harder to track down. We fetch variables in chunks of 5, so previously this test just had to wait for one variable fetch. But now there were suddenly six variables, so it needed two fetches.

What actually triggered the extra variable was an new 'import sys' that we added just a couple of days ago here for telemetry:
https://github.com/microsoft/vscode-jupyter/blob/ad0fd437952cc381c3d0952d97c13342d75ea8b4/src/client/datascience/jupyter/kernels/helpers.ts#L297

So we are doing an extra import here, and we usually try to hide our extra imports behind _VSCODE_ prefixes. But I don't believe that we need to add one here. We already import sys for some other things, so if you start an interactive window or a notebook, even several versions back, sys is already imported. We don't have the change the variable test expected results here as the sys is filtered out later by our variable exclude check that removes modules.

Reply:
Lol, So I literally i pushed it over the limit..